### PR TITLE
Bash cleanup

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
I'm happy to help with any more Bash scripting tasks as it comes along. I'm not an expert, but I do have a lot of experience.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+indent_style = space
+indent_size = 2
+
+# Unix line endings
+end_of_line = lf
+insert_final_newline = true
+
+[Makefile]
+indent_style = tab
```
Comment (openjck) [inline .editorconfig:14]: We don't have any Makefiles yet, but this is going to save us lots of confusion when we write our first one. 😆

**Comment by openjck** [comment]
This PR also adds a *.editorconfig* file, which will prevent small changes to ending lines and trailing spaces from polluting our commits. That said, the first time each file is edited, these types of changes will show up to adhere to this new *.editorconfig* standard.

Just be sure to install an EditorConfig plugin for your editor.

File: all [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```
Comment (hamilton) [inline all:1]: were these `git mv`'d or did you just rename these?

**Comment by openjck** [comment]
Good catch! I believed I was using the rename feature indirectly via `git add -A`, and it seems that Git correctly understood the rename in [that particular commit](https://github.com/graph-paper-org/graph-paper/pull/20/commits/84b5cd246b5a21cf66002ac288c52d0620f2d2a6) but it lost the post somewhere later on, thus the strange diff view. I'll look into this.

**Comment by openjck** [comment]
It looks like Git is tracking the rename correctly. Try checking out this branch and then running the following:

* `git log --follow all`
* `git log --follow create`

I'm not sure why GitHub's diff view is showing the files as being deleted and re-created under different names. Perhaps some heuristic gone haywire. GitHub is known to have some issues with renames and has a known issue where it [does not show history across renames](https://github.com/isaacs/github/issues/900), unfortunately.

**Comment by hamilton** [comment]
no biggie, thanks for the PR

**Pull request merged**